enzy_htp/structure/residue.py

```diff
@@ -301,6 +301,10 @@ def is_connected(self) -> bool:
     def has_init_charge(self) -> bool:
         """check if self has charge"""
         return math.prod([atom.has_init_charge() for atom in self.atoms])  
+    
+    def has_hydrogen(self) -> bool:
```

Comment:
We try to make each PR closely bind to its theme. So please have these fix in a different PR

Reply:
The modifications to files in `enzyhtp` directory are now unstaged.